scripts/abi_bindings.sh

```diff
@@ -54,6 +54,12 @@ forge build --force --extra-output-files abi
 
 contract_names=($CONTRACT_LIST)
 
+# Set the CGO flags to use the portable version of BLST
+#
+# We use "export" here instead of just setting a bash variable because we need
+# to pass this flag to all child processes spawned by the shell.
+export CGO_CFLAGS="-O -D__BLST_PORTABLE__"
```

Comment:
Can we instead put this in a `constants.sh` and `source` that file in all our scripts? This will be consistent with our other repos, e.g. `awm-relayer`, `subnet-evm`, `avalanchego`

Reply:
Yep, updated now. I updated `abi_binding.sh` and `e2e_test.sh` to first set `TELEPORTER_PATH` and then source the constants file from there, and removed unused constants.